doc/templates/documentation.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Refresh" content="0; url={{ pathto('index') }}" />
+  </head>
+  <body>
+    <p>Please follow <a href="{{ pathto('index') }}">this link</a>.</p>
```

Comment:
Maybe something like

"You will be automatically redirected to the [main page](link)"